addons/pos_self_order_sale/models/res_config_settings.py

```diff
@@ -8,6 +8,8 @@ class ResConfigSettings(models.TransientModel):
 
     @api.onchange("pos_self_order_kiosk")
     def _self_order_kiosk_change(self):
```

Comment:
Can you change the name of this function so that it matches the coding guidelines of the onchange ? (and change it in pos_self_order module aswell)

```suggestion
    def _onchange_self_order_kiosk(self):
```

Reply:
I will do it in another PR